liveblog/liveblog
# [LBSD-2774] - Make post types buttons visible from the start

### Purpose

This PR is an enhancement on the Editor where the block options are shown at the start and the user doesn't have to click the "ADD CONTENT HERE" button. 

### What has changed

- Block options are shown when the Editor is initially rendered.

### Steps to test

1. Navigate to the Editor's interface.
2. The block options should now be shown without having to click "Add Content Here"

_Note: This is only on initial render. Subsequently, the default behavior resume to allow the user to add content above or below existing content_

### Screenshots

| Original  | New |
| ------------- | ------------- |
| ![image](https://github.com/liveblog/liveblog/assets/14116447/07960668-ab22-4c59-ad35-c30b56bd25b3) | ![image](https://github.com/liveblog/liveblog/assets/14116447/b0bd8659-0449-4509-9b34-bd124d2c4db6)   |



Resolves: [LBSD-2774](https://sofab.atlassian.net/browse/LBSD-2774)

[LBSD-2774]: https://sofab.atlassian.net/browse/LBSD-2774?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
